src/test/java/BaseTest.java

```diff
@@ -1,11 +1,94 @@
 import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.chrome.ChromeOptions;
-import org.testng.annotations.BeforeSuite;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.*;
+
+import javax.swing.*;
+import java.time.Duration;
 
 public class BaseTest {
 
+    public WebDriver driver;
+    public WebDriverWait wait;
+    public Actions actions;
+    //public String url = "https://qa.koel.app/";
+
+    /*@DataProvider(name="NegativeLoginTestData")
+    public Object[][] getDataFromDataProvider(){
+        return new Object[][]{
+                {"invalidEmail@class.com","invalid"},
+                {"demo@class.com","invalid"},
+                {"",""},
+                {"invalidEmail@class.com","te$t$tudent"},
+        };
+    }*/
+
     @BeforeSuite
     static void setupClass() {
         WebDriverManager.chromedriver().setup();
     }
+
+    /*@BeforeMethod
+    public void launchBrowser(){
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--remote-allow-origins=*");
+        driver = new ChromeDriver(options);
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+        driver.manage().window().maximize();
+        navigateToPage();
+    }*/
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.